css/styles.css

```
@@ -0,0 +1,2212 @@
+  padding: var(--spacing-xxl);
+  border-radius: 4px;
+  transition: all 0.3s cubic-bezier(0.4, 0, 0.2, 1);
+  border: 1px solid var(--color-border);
+  background-color: var(--color-bg-card);
+  box-shadow: var(--color-shadow);
+  height: 100%;
+  width: 100%;
+  box-sizing: border-box;
+  margin: 0;
+  /* Add these new properties */
+  border: 1px solid transparent;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

border is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)